src/common/components/cards/card-kebab-menu-button.scss

```diff
@@ -1,86 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-@import '../../styles/colors.scss';
-@import '../../styles/common.scss';
+.kebab-menu-button {
+    margin-right: 8px;
 
-div.kebab-menu-callout {
-    box-shadow: 0 6.4px 14.4px $box-shadow-132, 0 1.2px 3.6px $box-shadow-108;
-    border-radius: 4px;
-    border-width: 0;
-
-    > div {
-        border-radius: 4px;
-    }
-}
-
-.kebab-menu {
-    background: $menu-background;
-    border: 1px solid $menu-border;
-
-    li {
-        background-color: $menu-background;
-    }
-
-    button {
-        i {
-            color: $primary-text;
-        }
-
-        &:hover {
-            background-color: $menu-item-background-hover;
-            @media screen and (forced-colors: active) {
-                background-color: inherit;
-            }
-        }
-
-        &:active {
-            background-color: $menu-item-background-active;
-            color: $always-black;
-
-            i {
-                color: $always-black;
-            }
-        }
-    }
-}
-
-button.kebab-menu-button {
-    width: 34px;
-    height: 32px;
-    margin: 8px;
-    border-radius: 2px;
-    background: transparent;
-
-    svg {
-        stroke: $primary-text;
-        @media screen and (forced-colors: active) {
-            fill: ButtonFace !important;
-            stroke: ButtonText !important;
-        }
-    }
-
-    &:hover {
-        background-color: $menu-item-background-hover;
-    }
-
-    &:active,
-    &[aria-expanded='true'] {
-        background-color: $menu-item-background-active;
-        color: $always-black;
-
-        svg > circle {
-            stroke: $always-black;
-        }
-        @media screen and (forced-colors: active) {
-            color: inherit;
-
-            svg > circle {
-                stroke: inherit;
-            }
-        }
+    :global(.ms-Icon) {
+        font-size: 18px; // Increase kebab icon size
     }
 
-    > span {
-        justify-content: center;
+    &:hover,
+    &:global(.is-expanded) {
+        background-color: #edebe9;
```

Comment:
Note: It's not clear to me why these styles aren't cascading from [FluentUI button](https://developer.microsoft.com/en-us/fluentui#/controls/web/button). Without this line the background color is set to `transparent`.

Reply:
Note to self: swap for a color variable.